gips/data/merra/merra.py

```diff
@@ -89,7 +89,8 @@ class merraAsset(Asset):
     }
 
     _bandnames = ['%02d30GMT' % i for i in range(24)]
-    _asset_re_pattern = "MERRA2_\d\d\d\.{name}\.%04d%02d%02d\.nc4"
+    _asset_re_pattern = 'MERRA2_\d\d\d\.%s\.\d{4}\d{2}\d{2}\.nc4'
```

Comment:
I know that the `format()` method is preferred, but that plays havoc with the regex repeat syntax.